Refs #32219 -- Added admin model inline tests for verbose names.

Siburg opened the pull request:
Add tests for verbose names and verbose plural names of inline models, depending on the values for `verbose_name` and/or `verbose_name_plural` of their parent model.

This precedes https://github.com/django/django/pull/13710, which has these tests as amended to reflect the updated functionality in that PR.

felixxm commented:
@Siburg Thanks :+1: I removed `test_verbose_parent` because it's a regression test what we will add in #13710. I also changed to inline classes and re-used existing models to show you what I asked in previous comments.

Siburg commented:
Cool. Thank you. I'm delighted if you get it to work the way you wanted. I
trust you can take it further and merge when you are comfortable.
Cheers, Jelle

On Wed, 8 Sept 2021 at 21:27, Mariusz Felisiak ***@***.***>
wrote:

> @Siburg <https://github.com/Siburg> Thanks 👍 I removed
> test_verbose_parent because it's a regression test what we will add in
> #13710 <https://github.com/django/django/pull/13710>. I also changed to
> inline classes and re-used existing models to show you what I asked in
> previous comments.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/django/django/pull/14838#issuecomment-915071812>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGZIFVIKXRILELEGJLVI55TUA4T7ZANCNFSM5DRZAO4Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
